.drone.yml

```diff
@@ -26,7 +26,7 @@ steps:
 
 - name: stage-binaries
   pull: default
-  image: rancher/dapper:1.11.2
+  image: rancher/dapper:v0.6.0
   commands:
   - "cp -r ./bin/* ./package/"
```

Comment:
@superseb this needs to be fixed as well. In 1.5 where it is merged, CI for tags is failing due to this step. The package dir is deleted and its trying to copy files into that dir.

Reply:
done, thanks!